tests/mtouch/MTouch.cs

```diff
@@ -3113,6 +3113,10 @@ public void TestAllLoad ()
 			using (var mtouch = new MTouchTool ()) {
 				mtouch.CreateTemporaryApp ();
 				mtouch.GccFlags = "-all_load";
+				// the test app doesn't depend on any BCL assembly that would cause mtouch to
+				// pull in GSS, but -all_load will cause all members of libmono-native to be
+				// loaded so we need to manually add GSS.framework or we'd get undefined symobls
+				mtouch.GccFlags += " -framework GSS";
```

Comment:
I think mtouch should add a reference to the GSS framework instead, since customers may use `-all_load`, and they shouldn't have to add additional mtouch arguments just because our libraries need it.

Reply:
yes, it's not a test issue. In fact the test shows it will affect existing projects (even if we dislike `-all_load` we cannot wish it to _unexist_)